docs/navigating-compliance.md

```diff
@@ -142,7 +144,7 @@ Please note the following about the compliance details page.
 
 # Supported Platforms
 
-Platforms support technically come from the options provided by nornir_nautobot. However, for reference, the valid slug's of the platforms are provided in the [FAQ](./FAQ.md).
+Platforms support technically come from the options provided by netutils. However, for reference, the valid slug's of the platforms are provided in the [FAQ](./FAQ.md).
```

Comment:
```suggestion
Platforms support technically come from the options provided by [netutils](https://github.com/networktocode/netutils). However, for reference, the valid slug's of the platforms are provided in the [FAQ](./FAQ.md).
```

Reply:
It's actually both, I have updated to reflect that, thanks for pointing out.